src/libFLAC/stream_encoder.c

```diff
@@ -1146,6 +1092,88 @@ static FLAC__StreamEncoderInitStatus init_stream_internal_(
 	encoder->protected_->seektable_offset = 0;
 	encoder->protected_->audio_offset = 0;
 
+	FLAC__bitwriter_release_buffer(encoder->private_->frame);
+	FLAC__bitwriter_clear(encoder->private_->frame);
+
+#if FLAC__HAS_OGG
+	/* reorder metadata if necessary to ensure that any VORBIS_COMMENT is the first, according to the mapping spec */
+	if(is_ogg && 0 != encoder->protected_->metadata && encoder->protected_->num_metadata_blocks > 1) {
```

Comment:
This fails because `is_ogg` is undefined. I believe it should be `encoder->private_->is_ogg`